installation.js

```diff
@@ -2,12 +2,23 @@ const {app} = require('./app.js')
 const {git} = app
 const {execSync} = require('child_process')
 const {pathExists, rimraf} = app.utils
+const logger = require('electron-timber')
+const mainLogger = logger.create({name: 'MainBrain'})
 
 const ipc = require('electron-better-ipc')
 
 const installEditorDependencies = (reinstall) => {
   app.notify(`${reinstall ? 'Reinstalling' : 'Installing'} editor dependencies`)
-  execSync(`. ${app.paths.nvs}/nvs.sh && nvs add 10.11 && nvs use 10.11 && cd ${app.paths.editor} && npm install`)
+  try {
+    mainLogger.log('Doing a log thing')
+    const sortIt = execSync(`. ${app.paths.nvs}/nvs.sh && nvs add 10.11`)
+    mainLogger.log('Doing a log thing part deux')
```

Comment:
Can remove?

Reply:
yes - obviously this needs a total and thorough rewrite. But that was just to check whether I was having network issues or nvs had just stopped working